### ISSUE TITLE
Heads don't spawn properly in plugins

### ISSUE THREAD

**Issue by TheAppleGamer** [iTXTech/Genisys#2086]
#### Issue description

When you use a plugin that spawns a head, it pretty much messes up on placing the head, as you can see between the two pictures below, there is the difference between PE and PC head spawning.
#### Steps to reproduce the issue

Try using the following code: `    public function setHead(Player $player){
      $level = $player->getLevel();
      $x = $player->getFloorX();
      $y = $player->getFloorY();
      $z = $player->getFloorZ();
      $pos = $player->getPosition();
      $level->setBlock($pos->add(0, 1, 0), Block::get(Block::SKULL_BLOCK), true, true);
      $level->setBlock($pos, Block::get(Block::NETHER_BRICK_FENCE));
      $chunk = $level->getChunk($x >> 4, $z >> 4);
      $nbt = new CompoundTag("", [
          new StringTag("id", Tile::SKULL),
          new IntTag("x", $x),
          new IntTag("y", $y+1),
          new IntTag("z", $z),
          new ByteTag("SkullType", SkullBlock::STEVE_HEAD),
          new ByteTag("Rot", floor(($player->yaw * 16) + 0.5))
      ]);
      Tile::createTile("Skull", $chunk, $nbt);
      $t = $level->getTile(new Vector3($x, $y+1, $z));
      $level->addTile($t);

    }`
#### OS and versions

* Genisys: c4407304
* PHP: 7.0.10
* Server OS: Windows 10
* Game version: 0.15.10

#### Crashdump, backtrace, memory dumps, plugins list or other files

http://imgur.com/a/2de8m
Sorry bad quality: http://imgur.com/a/0cfXl

**Comment by MCPEGamerJPatGitHub** [comment]
That's a MCPE bug, default heads work with fences. But as custom skin heads aren't actually in MCPE, it glitches out as an entity

**Comment by TheAppleGamer** [comment]
It does that without custom skin heads, I should have noted that

**Comment by IrishPacks** [comment]
That is a default head, but skinned as a custom one via texture packs. So no it doesn't glitch as an entity, but placing it via code is weird. So I've had this happen as well.

**Comment by JackNoordhuis** [comment]
Does this issue still occur and does it occur on other blocks as well?

**Comment by IrishPacks** [comment]
Yes it still occurs, it occurs with only this tile. The skull spawns in the middle of the block instead of the bottom if spawned by plugin.
